upmc-enterprises/cda2fhir
# Feature/linting smells

#### What does this PR do?

It's for Henry to look at linting changes I was asked to make

#### Related JIRA tickets:

SAR-47

#### How should this be manually tested?

If you have SONARlint, run it through there. 

Also make sure the code works as intended.
